flasksrc/seed_data/toynet_quizzes.sql

```diff
@@ -0,0 +1,161 @@
+
+
+INSERT INTO toynet_quizzes(quiz_id) VALUES(4001);
```

Comment:
Don't forget to drop all tables at the beginning of your SQL file so you're not creating duplicates across different runs and this database will be static. Not ideal for production, but we can take that back out later. Mentioned under "Extra Context" in the [original ticket](https://trello.com/c/VUCo4o1d).